test/capital-project/capital-project.repository.mock.ts

```diff
@@ -16,6 +17,27 @@ import {
 } from "src/gen";
 
 export class CapitalProjectRepositoryMock {
+  findCapitalProjectsMocks = Array.from(
+    Array(4),
+    (_) =>
+      generateMock(findCapitalProjectsRepoSchema, {
+        stringMap: {
+          minDate: () => "2018-01-01",
+          maxDate: () => "2045-12-31",
+        },
+      })[0],
+  );
+
+  async findCapitalProjects() {
+    return {
+      limit: 20,
+      offset: 0,
+      total: 4,
+      order: "managingCode, capitalProjectId",
+      capitalProjects: this.findCapitalProjectsMocks,
+    };
```

Comment:
The repository mock reflects what the database query would return, which does not include the order and pagination metadata. (These data are added by the service)

```suggestion
    return this.findCapitalProjectsMocks;
```